### PR TITLE
chore: agenda list table and action

### PR DESCRIPTION
**Pull request by Ibwedagama**
#### Related
- [S27A1 - View List Agenda
](https://airtable.com/app7SdZInMN1pG6ZL/tblLy5A3qYF19fj1u/viwDUthwzD6mc5jka/recTTgwhlTGM1QRPw?blocks=hide)

#### Changes
- create agenda table component
- create agenda table action
- add `vue-clickaway` package

#### Note
- table currently only shows dummy data
- table actions functionality are not implemented yet

#### Preview
Peek 2022-01-05 11-51

### Evidence
title: create agenda list table and action
project: Portal Jabar
participants: @Ibwedagama @yoslie @maulanayuseph @maruf12 @adzharamrullah @bangunbagustapa

### CODE REVIEW

**Review by adzharamrullah — APPROVED** [review]

**Review by yoslie — APPROVED** [review]

File: src/components/Agenda/AgendaTable.vue [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+export default {
+  name: 'AgendaTable',
+  components: {
+    AgendaTableAction,
+  },
+  data() {
+    return {
+      tableHeader: AGENDA_TABLE_HEADER,
+      // TODO: Remoe this dummy data with real data
```
Comment (yoslie) [inline src/components/Agenda/AgendaTable.vue:27]: typo `remoe `🤣

**Review by maruf12 — APPROVED** [review]

**Review by maulanayuseph — APPROVED** [review]

File: src/components/Agenda/AgendaTable.vue (hunk 1 above)
Comment (Ibwedagama) [reply src/components/Agenda/AgendaTable.vue:27]: sorry for the typo, I think it's okay for now 😆 @yoslie

File: src/components/Agenda/AgendaTableAction.vue [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+import { directive as onClickaway } from 'vue-clickaway';
+
+export default {
+  directives: {
+    onClickaway,
+  },
+  props: {
+    item: {
+      type: Object,
+      default: () => {},
```
Comment (bangunbagustapa) [inline src/components/Agenda/AgendaTableAction.vue:68]: You forgot the parentheses here

Comment (Ibwedagama) [reply src/components/Agenda/AgendaTableAction.vue:68]: I see sorry about that @bangunbagustapa I'll fix it asap

Comment (Ibwedagama) [reply src/components/Agenda/AgendaTableAction.vue]: fixed [here](https://github.com/jabardigitalservice/portal-jabar-cms/pull/17/commits/8ea23da4c6f93932a3c051bcdb93258671e7e241) thanks @bangunbagustapa 👍🏼

**Review by bangunbagustapa — APPROVED** [review]

**Pull request merged**